sig-services/service-requirements/contract-duration.md

```diff
@@ -0,0 +1,26 @@
+---
+title: contract duration
+persona: service consumer, service provider/owner
+phase: service-operational
+---
+
+This requirement states that a service should have a defined contract duration. Contract duration states the start and the end of the subscription to the service or the conditions under which a subscription is terminated.
+
+If the contract duration states the end date of the subscription it should also state whether the subscription can be extended. If it can, it should also state if the subscription is going to auto extend without the consumers explicit acknowledgement.
+
+It should also state under what circumstances the service provider can involuntarily terminate the consumer's subscription (e. g., breach of Terms and Conditions). Additionally the service consumer should be advised how to end the service subscription on his own just at any time, especially in a case when the contract specifies a distinct end date of a subscription and what terms apply in such scenario.
+
+## Requirements
+
+- Create a file/page which defines the contract duration
+- Define cases when the consumer will be involuntarily unsubscribed from the service
+- Define that the consumer can unsubscribe from the service on his own at any time
+- Define whether the service subscription can be extended and whether it auto extends
+- Contract duration is not a simple link to lengthy Terms and Conditions document but rather a simple list that links back to the Terms and Conditions
```

Comment:
What is this point trying to convey? Would there be a separate Terms and Conditions document that the contract document should refer to?

Reply:
Resolved in a meeting.